src/Shared/ProcessExtensions.cs

```diff
@@ -14,100 +11,31 @@ internal static class ProcessExtensions
     {
         public static void KillTree(this Process process, int timeoutMilliseconds)
         {
+#if NETCOREAPP
+            process.Kill(entireProcessTree: true);
+#else
             if (NativeMethodsShared.IsWindows)
```

Comment:
I'm guessing you left this in rather than just having the windows code because you want the SupportedOSPlatform check to be happy? That seems reasonable but took me a second 🙂